.github/workflows/main.yml

```diff
@@ -0,0 +1,46 @@
+name: Build Release Image
+
+on:
+  workflow_dispatch:
+  # Run every day at 5:40.
+  schedule:
+    - cron:  '40 5 * * *'
+jobs:
+  buildflow:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+      - name: Determine latest un-imaged version
+        run: |
+          python get_latest_new_release.py
+          echo "version=$(cat _version.txt)" >> $GITHUB_ENV
```

Comment:
I understand why you did it like this, but I'd prefer if the script didn't create a file and instead just printed the version to `stdout`, which you can then pipe to the env directly.

But since the info/debug prints are useful, they should stay, but end in `stderr` instead (which will be ignored by the pipe).